tests/gmsh/gmsh_api_01.cc

```diff
@@ -22,8 +22,10 @@
 #include "../tests.h"
 
 int
-main()
+main(int argc, char **argv)
 {
+  // gmsh might be build with mpi support enabled.
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv);
```

Comment:
Why is this necessary? These tests aren't run under MPI, are they?

Reply:
@bangerth When you build gmsh with MPI support then gmsh is running under an MPI context.
Test failure: https://cdash.dealii.org/test/22159767
```
*** The MPI_Comm_rank() function was called after MPI_FINALIZE was invoked.
*** This is disallowed by the MPI standard.
*** Your MPI job will now abort.
[testsuite:1942435] Local abort after MPI_FINALIZE started completed successfully, but am not able to aggregate error messages, and not able to guarantee that all other processes were killed!
```

Reply:
I see. Thanks for the clarification!